### DISCUSSION TITLE
741

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#923] [Comments]
# 741

최근 미야자키 하야오 감독님의 "그대들은 어떻게 살 것인가" 영화가 개봉된다는 소식을 들었다. 제목이 너무 낯익어서 찾아봤더니 미야자키 하야오 감독님이 이 책을 오마주해서 영화를 만들었다고 한다. 대학생일때 도서관에서 빌려서 읽었던 책인데, 당시에는 책 내용이 너무 가식적이라고 생각했었다. 중2병이 늦게 와서 그랬는지 모르겠지만, 그땐 되게 시니컬했고 그만큼 이런 내용에 거부감이 있었다. 그래서 책을 읽고나서는 "이런 책을 왜 사람들이 좋아하는지 모르겠다" 라는 생각을 했었다. 그러다가 취업 준비, 이직 준비, 여러 주변의 변화 등이 있었는데, 이런 변화들이 있을때마다 이 책의 내용이 떠올랐다. 그리고 나는 어떤 어른이 되고 싶은지, 어떤 사람이 되고 싶은지에 대한 생각을 하게 되었다. 이 책을 보면 주..

https://jojoldu.tistory.com/741

**Comment by chrishoonS** [comment]
시중에 즐비한 자아 성찰 관련 도서들과는 달리 깊이 있는 고찰을 엿볼 수 있네요. 당연한 것을 생각한다라는 말이 정말 마음에 와닿는것 같습니다. 좋은 책 소개, 좋은 글 감사합니다!

**Comment by haeri2** [comment]
남겨주신 문장들이 꼭 읽어봐야겠다는 생각을 들게 하네요! 책 추천 감사드립니다 :>
